docs/en/query_language/alter.md

```diff
@@ -495,4 +495,53 @@ A mutation query returns immediately after the mutation entry is added (in case
 
 Entries for finished mutations are not deleted right away (the number of preserved entries is determined by the `finished_mutations_to_keep` storage engine parameter). Older mutation entries are deleted.
 
+## ALTER USER {#alter-user-statement}
+
+Changes ClickHouse user accounts.
+
+### Syntax {#alter-user-syntax}
+
+``` sql
+ALTER USER [IF EXISTS] name
+    [RENAME TO new_name]
+    [IDENTIFIED [WITH {PLAINTEXT_PASSWORD|SHA256_PASSWORD|DOUBLE_SHA1_PASSWORD}] BY {'password'|'hash'}]
+    [[ADD|REMOVE] HOST {LOCAL | NAME 'name' | NAME REGEXP 'name_regexp' | IP 'address' | LIKE 'pattern'} [,...] | ANY | NONE]
```

Comment:
RBAC-8 replaces `REMOVE` here to `DROP`.

Reply:
```suggestion
    [[ADD|DROP] HOST {LOCAL | NAME 'name' | NAME REGEXP 'name_regexp' | IP 'address' | LIKE 'pattern'} [,...] | ANY | NONE]
```